tests/virt/utils.py

```
@@ -366,3 +374,110 @@ def validate_machine_type(vm, expected_machine_type):
+    ):
+        yield
+
+
+def fetch_gpu_device_name_from_vm_instance(vm):
+    devices = vm.vmi.instance.spec.domain.devices
+    if devices.get("gpus") and devices.gpus:
+        return devices.gpus[0].deviceName
+    elif devices.get("hostDevices") and devices.hostDevices:
+        return devices.hostDevices[0].deviceName
+    else:
+        raise ValueError(f"No GPU devices found in VM {vm.name}")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)